draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1745,7 +1748,8 @@ Example-Dates: "Sat, 04 May 1996", "Wed, 14 Sep 2005"
    that allows both unquoted (token) and quoted (quoted-string) syntax for
    a parameter value (<xref target="parameter"/>). Use of common syntax
    allows recipients to reuse existing parser components. When allowing both
-   forms, the meaning of a parameter value ought to be the same whether it
+   forms, the meaning of a parameter value (with no internal commas)
```

Comment:
What is the "(with no internal commas)" about?

Reply:
no idea, will remove this part

Reply:
You can't use the token syntax if there are commas in the parameter value.  If there are such "internal commas" in the parameter value, then only the quoted-string syntax can be used for that particular value, regardless of whether or not the parameter in question allows both token and quoted-string form.
So this statement about "meaning ought to be the same whether it was received as [...]" is nonsensical for a parameter value that has internal commas.

Reply:
No, the statement is true for syntactically valid tokens and quoted-strings.